src/main/java/fi/fmi/avi/model/SpaceWeatherAdvisory/SpaceWeatherAdvisoryAnalysis.java

```diff
@@ -0,0 +1,22 @@
+package fi.fmi.avi.model.SpaceWeatherAdvisory;
+
+import java.util.List;
+import java.util.Optional;
+
+import fi.fmi.avi.model.PartialOrCompleteTimeInstant;
+
+public interface SpaceWeatherAdvisoryAnalysis {
+    PartialOrCompleteTimeInstant getTime();
+
+    Optional<Type> getAnalysisType();
+
+    Optional<List<SpaceWeatherRegion>> getRegion();
+
+    boolean isNoPhenomenaExpected();
+
+    boolean isNoInformationAvailable();
```

Comment:
Maybe we should consider a common `enum NilReason`, containing all NIL reasons specified in WMO [Register: Code Table D-1: Nil reasons](https://codes.wmo.int/common/nil). And then, replacing mutually exclusive `boolean isNoPhenomenaExpected()` and `boolean isNoInformationAvailable()` with a `NilReason` field?

Reply:
Effects to TAF, METAR and SPECI must be resolved before trying this out. The booleans mentioned are now replaced by an enum. Therefore this idea won't be implemented right now.